features/step_definitions/web.rb

```diff
@@ -240,3 +240,33 @@
 Given /^I close the current browser$/ do
   browser.finalize
 end
+
+Given /^I check all relatedObjects of clusteroperator "(.*?)" are shown/ do |clusteroperator_name|
+  ensure_admin_tagged
+  #get relatedObjects of clusteroperator
+  relatedObjectsArray = cluster_operator(clusteroperator_name).related_objects
+  relatedObjectsArray.each do |related_object|
+    if !related_object.key?('name') or related_object['name'] == ""
+      next
```

Comment:
IMO, we always need a name for whatever the resources it is.
A resource without name or null name sounds like a bug.

Reply:
Yeah that's also my assumption however not every resource the cluster operators set has name field, for example 
https://github.com/openshift/cluster-ingress-operator/blob/48dd8e15152c8c9638a09de8c489131c85854253/pkg/operator/controller/status/controller.go#L116~L131
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/starter.go#L195~L208

Reply:
Ack. Make sense now. They want all resources of the specific kind, e.g, scc